docs/admin-guides/managing-optimizers.md

```diff
@@ -89,6 +89,19 @@ containers:
       flink-conf.kubernetes.service-account: flink                                   # Service account that is used within kubernetes cluster.
 ```
 
+An example for flink session mode:
+
+```yaml
+containers:
+  - name: flinkContainer
+    container-impl: com.netease.arctic.server.manager.FlinkOptimizerContainer
+    properties:
+      target: session                                                                # Flink run in session cluster
+      job-uri: "local:///opt/flink/usrlib/optimizer-job.jar"                         # Optimizer job main jar
+      ams-optimizing-uri: thrift://ams.amoro.service.local:1261                      # AMS optimizing uri 
+      export.FLINK_CONF_DIR: /opt/flink/conf/                                        # Flink config dir, flink-conf.yaml should e in this dir, contains the rest connection parameters of the session cluster
```

Comment:
session cluster (jobmanager url) must be set in flink-conf?

Reply:
Not necessarily. If it is a high-availability cluster, such as zk, you can directly specify the zk configuration. There is no need to configure the jobmanager address, such as:
```
high-availability: zookeeper
high-availability.zookeeper.quorum: xxx:2181
high-availability.zookeeper.path.root: /flink
high-availability.cluster-id: amoro-optimizer-cluster
high-availability.storageDir: hdfs://xxx/xxx/xxx
```

Reply:
I suggest adding the parameters that must be set in session mode to the example. For example, jobmanager.url or zk-related settings

Reply:
Should optimizer-job.jar be added to the Flink Path when a session is started in session mode?

Reply:
> I suggest adding the parameters that must be set in session mode to the example. For example, jobmanager.url or zk-related settings

+1, done.

Reply:
> Should optimizer-job.jar be added to the Flink Path when a session is started in session mode?

sry. I don’t quite understand the meaning of adding `optimizer-job.jar` to the flink path. Can you explain it in a little more detail?